TemplatesFiles/blender-2.8/UI/customizable_sidebar_tab.py

```diff
@@ -0,0 +1,105 @@
+bl_info = {
+    "name": "Cuztomizable Sidebar Tab",
+    "author": "Your Name Here",
+    "version": (1, 0),
+    "blender": (3, 2, 0),
+    "location": "View3D > Sidebar",
+    "description": "Adds a new Mesh Object",
+    "warning": "",
+    "doc_url": "",
+    "category": "Add Mesh",
+}
+
+
+import bpy
+from bpy.types import Operator, Panel, AddonPreferences
+from bpy.props import FloatVectorProperty, StringProperty
+from bpy_extras.object_utils import AddObjectHelper, object_data_add
+from mathutils import Vector
```

Comment:
Le PEP8 demande de faire les imports du code avant d'initialiser des variables, ce n'est en aucun point problématique, mais c'est une règle de code.

Reply:
si je comprends bien la [ doc python de blender](https://wiki.blender.org/wiki/Style_Guide/Python), ils font des exceptions de ce côté là?

> We often put imports within functions body which is not pep8 compliant.
This is done to speed up Blender's startup times to avoid loading in many files and libraries which the user may never access.

en tout cas le bl_info, ils le mettent toujours en premier dans les addons de la BF.

Reply:
Han exact, je n'avais jamais vu ça dans le wiki, je suis surtout sur l'api. Je valide la pull request.
En vrai il faudrait faire remonter l'info dans la doc de l'api.

Reply:
Sinon il y a aussi l'utilisation des double quotes qui est normalement pour les paths, les strings normales sont utilisé pour le reste.